### ISSUE TITLE
Heartbeat messages not getting through

### ISSUE THREAD

**Issue by patriknw** [akka/akka#21583]
See https://github.com/real-logic/Aeron/issues/281
Not sure if it's an Aeron problem or if we are doing something stupid.

The branch I used for testing and debugging this is https://github.com/akka/akka/tree/artery-app

Test app started with:
```
rm -rf target/aeron-LotsOfDataBot2/ target/LotsOfDataBot2.afr

sbt -Dakka.remote.artery.canonical.hostname=$HOSTNAME -Dakka.cluster.seed-nodes.0=akka://ClusterSystem@ip-172-31-10-77:25520 -Dakka.cluster.seed-nodes.1=akka://ClusterSystem@ip-172-31-8-204:25520 -J-Xms3g -J-Xmx3g -J-XX:+PrintGCDetails -J-XX:+PrintGCTimeStamps -J-XX:MaxDirectMemorySize=256m -J-XX:-HeapDumpOnOutOfMemoryError -Dsbt.log.noformat=true -Dakka.remote.artery.log-sent-messages=on -Dakka.remote.artery.log-received-messages=on -Dakka.loglevel=DEBUG "akka-distributed-data-experimental/test:runMain akka.cluster.ddata.LotsOfDataBot2 25520" > out35.txt 2>&1
```

First on node1, and after ~20 seconds on node2.
In logs there is unexpected unreachability when the issue happens, which is not every time.

AeronStat is useful:

```
java -cp /home/ubuntu/aeron-lib/aeron-samples-1.0.1.jar:/home/ubuntu/aeron-lib/Agrona-0.5.4.jar:/home/ubuntu/aeron-lib/aeron-driver-1.0.1.jar:/home/ubuntu/aeron-lib/aeron-client-1.0.1.jar -Daeron.dir=target/aeron-LotsOfDataBot2/ io.aeron.samples.AeronStat type=[1-9]
```

**Comment by patriknw** [comment]
This is seems to be fixed in upcoming Aeron 1.0.2. I can't reproduce it with latest master of Aeron as of today.

**Comment by patriknw** [comment]
The dependency is updated in upcoming Akka 2.4.12, but you can take advantage of Aeron 1.0.2 immediately with Akka 2.4.11 by replacing the dependencies:

```
"io.aeron"                      % "aeron-driver"                 % "1.0.2",
"io.aeron"                      % "aeron-client"                 % "1.0.2"
```
